common/persistence/client/factory.go

```diff
@@ -125,6 +127,7 @@ var storeTypes = []storeType{
 // The objects returned by this factory enforce ratelimit and maxconns according to
 // given configuration. In addition, all objects will emit metrics automatically
 func NewFactory(
+	persistenceMaxQPS dynamicconfig.IntPropertyFn,
 	cfg *config.Persistence,
```

Comment:
nit: probably move maxQPS after cfg? 

Just my personal preference, feel free to ignore.

Reply:
I agree